r/R/dataset-format.R

```diff
@@ -33,6 +33,10 @@
 #'   * "tsv", equivalent to passing `format = "text", delimiter = "\t"`
 #' * `...`: Additional format-specific options
 #'
+#'   `format = "csv"``:
+#'   * `compression`: Assume CSV files have been compressed with this codec.
+#'   Any options from [CsvParseOptions] may also be passed.
+#'
 #'   `format = "parquet"``:
```

Comment:
```suggestion
#'   `format = "parquet"`:
```
Should fix this while we're here.